src/server/svr_func.c

```diff
@@ -1653,12 +1653,12 @@ static int svr_oldstyle[] = {
 	(int)SRV_ATR_max_running,
 	(int)SRV_ATR_MaxUserRun,
 	(int)SRV_ATR_MaxGrpRun,
-	(int)SVR_ATR_MaxUserRes,
-	(int)SVR_ATR_MaxGroupRes,
-	(int)SVR_ATR_MaxUserRunSoft,
-	(int)SVR_ATR_MaxGrpRunSoft,
-	(int)SVR_ATR_MaxUserResSoft,
-	(int)SVR_ATR_MaxGroupResSoft,
+	(int)SRV_ATR_MaxUserRes,
```

Comment:
Why the change from SVR to SRV?

Reply:
Half the variables are prepended with SVR and half are SRV, so went one way..

Reply:
Ahh, yes consistency is good.  I personally would have voted for SVR, but I guess it doesn't really matter.  What is the rest of the code doing (function names and the like)?

Reply:
actually, i think u are right? SVR Is more prominent and not SRV, i will switch to SVR for better consistency with rest of the code